clients/drcachesim/tools/view.cpp

```diff
@@ -493,6 +493,10 @@ view_t::parallel_shard_memref(void *shard_data, const memref_t &memref)
             std::cerr << "<marker: vector length " << memref.marker.marker_value
                       << " bytes>\n";
             break;
+        case TRACE_MARKER_TYPE_UNCOMPLETED_INSTRUCTION:
+            std::cerr << "<marker: uncompleted instruction, encoding 0x" << std::hex
+                      << memref.marker.marker_value << std::dec << ">\n";
```

Comment:
Is it worth trying to print in the encoding style of the ISA (separate bytes for x86, eg..)?  Since it may be incomplete: maybe not.  Yeah I would just leave it like this.  How about adding a comment though?